isotovideo

```diff
@@ -112,8 +113,15 @@ use commands;
 
 $bmwqemu::vars{BACKEND} ||= "qemu";
 
-# Load the main.pm from the casedir checked by the sanity checks above
-require $bmwqemu::vars{CASEDIR}."/main.pm";
+# Try to load the main.pm from one of the following in this order:
+#  - product dir
+#  - casedir
+#
+# This allows further structuring the test distribution collections with
+# multiple distributions or flavors in one repository.
+diag "product dir : $bmwqemu::vars{PRODUCTDIR}";
```

Comment:
This will produce undefined variable warning when not set. Why not switch it with L123?